README.md

```diff
@@ -8,16 +8,18 @@ The GDC MAF tool aggregates aliquot-level MAFs, which originate from one tumor-n
 
 Ensemble aliquot-level MAFs can be queried at the GDC Data Portal (https://portal.gdc.cancer.gov/repository) using the following filters along with the project of your choice:
 
-* Data Format: maf
+
+* Data Type: Masked Somatic Mutation
 * Workflow Type: Aliquot Ensemble Somatic Variant Merging and Masking
+* Data Format: maf
 
 Alternatively, this data set can be accessed at the following link:
 https://portal.gdc.cancer.gov/repository?filters=%7B%22op%22%3A%22and%22%2C%22content%22%3A%5B%7B%22op%22%3A%22in%22%2C%22content%22%3A%7B%22field%22%3A%22files.analysis.workflow_type%22%2C%22value%22%3A%5B%22Aliquot+Ensemble+Somatic+Variant+Merging+and+Masking%22%5D%7D%7D%2C%7B%22op%22%3A%22in%22%2C%22content%22%3A%7B%22field%22%3A%22files.data_format%22%2C%22value%22%3A%5B%22maf%22%5D%7D%7D%5D%7D
```

Comment:
Does the portal link also need to change?

Reply:
Yes!  Good catch!